sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/test/TestTableProvider.java

```diff
@@ -186,13 +198,21 @@ public BeamTableStatistics getTableStatistics(PipelineOptions options) {
 
       PCollection<Row> result = withAllFields;
       if ((options == PushDownOptions.FILTER || options == PushDownOptions.BOTH)
-          && !(filters instanceof DefaultTableFilter)) {
-        throw new RuntimeException("Unimplemented at the moment.");
```

Comment:
If the filter type is something unexpected this should throw an exception.

Reply:
Throwing exception when encountering an unexpected `TableFilter`.